resources/tests.mjs

```diff
@@ -332,6 +332,52 @@ Suites.push({
     ],
 });
 
+Suites.push({
+    name: "Editor-CodeMirror",
+    url: "tentative/editors/dist/codemirror.html",
+    async prepare(page) {
+        page.querySelector("#create").click();
+        await page.waitForElement("#create[disabled]");
+    },
+    tests: [
+        new BenchmarkTestStep("Big", (page) => {
+            page.querySelector("#big").click();
+            page.querySelector("#layout").click();
+        }),
+        new BenchmarkTestStep("Highlight", (page) => {
+            page.querySelector("#highlight").click();
+            page.querySelector("#layout").click();
+        }),
+        new BenchmarkTestStep("Scroll down", (page) => {
+            page.querySelector("#scroll").click();
+            page.querySelector("#layout").click();
+        }),
+    ],
+});
+
+Suites.push({
+    name: "Editor-TipTap",
```

Comment:
@rniwa for some reason this particular workload seems slow on WebKit, at least on my laptop. I don't think it's specific to how it's integrated within the harness (i.e. I also see jank loading https://tiptap.dev/examples/book), but if you see anything that looks wrong with the test itself let's discuss. Note the PR is only using the first chapter and not the entire book - I haven't tested what happens with much longer text.

Reply:
Yeah, I looked into it but I don't see anything obviously wrong.